integration_tests/src/management_api_tests.rs

```diff
@@ -0,0 +1,73 @@
+use std::{collections::HashMap, str::FromStr};
+
+use entities::api_req_params::GetAsset;
+use function_name::named;
+use nft_ingester::management_api::handlers::asset_fix::FixAccountAssetsRequest;
+use serial_test::serial;
+use solana_sdk::pubkey::Pubkey;
+use tokio_util::sync::CancellationToken;
+
+use super::common::*;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+#[named]
+async fn test_management_api_fix_account_assets() {
+    let name = trim_test_name(function_name!());
+    let setup = TestSetup::new_with_options(
+        name.clone(),
+        TestSetupOptions {
+            network: Some(Network::Mainnet),
+            clear_db: true,
+            well_known_fungible_accounts: HashMap::new(),
+        },
+    )
+    .await;
+    let account = Pubkey::from_str("124YCZJGFhyiB9fVjmQdqxGrmpFMSxekNndWCxRTftbP").unwrap();
+    // Start the management API
+    let cancellation_token = CancellationToken::new();
+    let metrics = setup.metrics.ingester_metrics.clone();
+    let rocks_storage = setup.rocks_db.clone();
+    let postgre_client = setup.db.clone();
+    let cloned_rpc_client = setup.client.clone();
+    let well_known_fungible_accounts = HashMap::new();
+    let port = 3333;
+    usecase::executor::spawn({
+        let cancellation_token = cancellation_token.child_token();
+        async move {
+            nft_ingester::management_api::start_management_api(
+                cancellation_token,
+                metrics,
+                rocks_storage,
+                postgre_client,
+                cloned_rpc_client,
+                well_known_fungible_accounts,
+                port,
+            )
+            .await
+        }
+    });
```

Comment:
i would advise against using our static executor (joinset) here. you're not performing graceful shutdown in tests anyways, so `tokio::task::spawn` is a better alternative here.